dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -42,10 +42,10 @@
 = render partial: 'levels/editors/sub_level_types', locals: {f: f}
 
 - if @level.is_a?(Weblab) || (@level.uses_droplet?) || @level.is_a?(Blockly) || @level.respond_to?(:disable_sharing)
-  %legend.control-legend{data: {toggle: "collapse", target: "#shareAndRemix"}}
+  %legend.control-legend{data: {toggle: "collapse", target: "#share_and_remix"}}
```

Comment:
i'm so sorry to make you change these again, but we use hyphens in CSS instead of underscores or camelcase (`#share-and-remix`)

Reply:
I made them consistent with other ones in these same files. Leaving consistent for now.